.env.example

```
@@ -1,3 +1,15 @@
+VITE_SPACETIMEDB_MODULE=
+VITE_SPACETIMEDB_HOST=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_SPACETIMEDB_HOST key should go before the VITE_SPACETIMEDB_MODULE key

(UnorderedKey)